### PR TITLE
Modified the appearance of the options for font and color, now it shows a button for preview and changing the formats

### PR DESCRIPTION
**Pull request by deaspo**
Modified the appearance of the options for font and color, now it shows a button for preview and changing the formats

### CODE REVIEW

File: instat/dlgOptions.Designer.vb [hunk 1]
```diff
@@ -31,46 +31,21 @@ Partial Class dlgOptions
...
-        Me.lblScriptSize = New System.Windows.Forms.Label()
-        Me.txtScriptSize = New System.Windows.Forms.TextBox()
-        Me.txtOutputSize = New System.Windows.Forms.TextBox()
-        Me.lblOutputsize = New System.Windows.Forms.Label()
-        Me.txtCommentsSize = New System.Windows.Forms.TextBox()
-        Me.lblCommentsSize = New System.Windows.Forms.Label()
+        Me.lblRScript = New System.Windows.Forms.Label()
+        Me.lblOutput = New System.Windows.Forms.Label()
+        Me.lblComments = New System.Windows.Forms.Label()
+        Me.Label1 = New System.Windows.Forms.Label()
```
Comment (dannyparsons) [inline instat/dlgOptions.Designer.vb]: What is Label1? This needs a proper name.

File: instat/dlgOptions.vb [hunk 2]
```diff
@@ -161,12 +139,10 @@ Public Class dlgOptions
         dlgFont.ShowColor = True
         dlgFont.MaxSize = 15
         dlgFont.MinSize = 8
+        dlgFont.Font = frmMain.clsRLink.fOutput
+        dlgFont.Color = frmMain.clsRLink.clrOutput
```
Comment (dannyparsons) [inline instat/dlgOptions.vb]: I think we shouldn't need to be getting these from frmMain. We can have default values to use the first time. Does the dialog keep the previous values the next time it opens?

Comment (deaspo) [inline instat/dlgOptions.vb]: Every time the dialog opens it picks the previous values as set in the Rlink, that's why I included line 142 and 143.

**Comment by deaspo** [comment]
The new request, addresses the two issues mentioned above. The default have been set in the clsRlink. Happy to discuss is needed.

File: instat/dlgOptions.vb (hunk 2 above)
Comment (dannyparsons) [inline instat/dlgOptions.vb]: There could be better ways to do this but this looks good for now. We can review it later if needed.

**Pull request merged**